.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of the change or the fix you've made. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please select atleast one
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Testing
+How was this change tested?
+
+## Documentation
+
+_Please list any documentation changes or additions. Please delete if not necessary._
+
+## Addtional Info
+
+_Please enter any additional information that could be helpful for the reviewers or just to document. Please delete if not necessary._
+
+## Checklist:
+
+- [ ] I have run swiftformat or other relevant tools to follow style guidelines.
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
CI should be able to detect this automatically right?

Reply:
Yes, it would. I wanted to add everything i could think of at first. We can remove checks that are not needed (probably like in this case) or add more if i am missing something.

Reply:
Yes, makes sense to rm this line.